.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Purpose
+
+Fixes:
+
+## Examples
+
+## Checklist
+- [ ] Linked to an issue
```

Comment:
There need to be something related to `read the guidelines of a PR`. Can you check other PR templates?